### PR TITLE
change special character in README

### PR DESCRIPTION
**Pull request by CWempe**
When copy-and-paste the filename `post‑install.txt` you get an error while installing, because the hyphen was written as a special character.

```
Copying /boot files in... cp: can't create '/boot/post?install.txt': Invalid argument
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ So don't copy and paste the defaults from below!
 
 The _installer-config.txt_ is read in at the beginning of the installation process, shortly followed by the file pointed to with `online_config`, if specified.
 There is also another configuration file you can provide, _post&#8209;install.txt_, and you place that in the same directory as _installer-config.txt_.
```
Comment (Mausy5043) [inline README.md]: Any reason why you didn't change L103 too?

**Comment by CWempe** [comment]
Oops. I didn't saw that.

Fixed it too, now.

**Comment by Mausy5043** [comment]
Thanks

**Comment by CWempe** [comment]
Changed PR to merge with v1.1.x

**Comment by diederikdehaas** [comment]
@Mausy5043 was right that it should be submitted against v1.1.x, but you should've rebased it as well.
Now this PR contains various commits which are already in v1.1.x (and because of that contains various conflicts.

Therefor kindly apply your changes on top of the current v1.1.x branch and submit a new PR for that

**Pull request closed without merging**